CardanoSharp.Wallet/Extensions/Models/Transactions/TransactionExtensions.cs

```diff
@@ -93,8 +93,11 @@ public static uint CalculateFee(this Transaction transaction, uint? a = null, ui
             return ((uint)transaction.Serialize().Length * a.Value) + b.Value;
         }
 
-        public static uint CalculateAndSetFee(this Transaction transaction, uint? a = null, uint? b = null)
+        public static uint CalculateAndSetFee(this Transaction transaction, uint? a = null, uint? b = null, int mockedVKeyWitnesses = 0)
```

Comment:
opinion: param name should maybe be something like expectedVKeyWitnessCount?
or numberOfVKeyWitnessesToMock?

while we are using the mock witnesses to perform the calc, i think the name isnt necessarily self explanatory, it could be misinterpreted as someone needing to put in the number of mock witnesses they have made already.